drivers/dai/intel/ssp/ssp.c

```diff
@@ -872,10 +871,53 @@ static void dai_ssp_pm_runtime_dis_ssp_power(struct dai_intel_ssp *dp, uint32_t
 #endif /* CONFIG_DAI_SSP_HAS_POWER_CONTROL */
 }
 
+#if defined(CONFIG_SOC_INTEL_ACE30_PTL)
+static inline size_t popcount64(uint64_t mask)
+{
+	size_t count = 0;
+
+	for (; mask != 0; count++) {
+		mask &= mask - 1;
```

Comment:
It'd be surprised if this does not exist somewhere in Zephyr already.

Also: this is not PTL specific; no need to guard this with an `#ifdef`?